### PR TITLE
Issue 4444

### PR DESCRIPTION
**Pull request by nbintertech**
connects #4444 

- Adding flue gas to Treasure Hunt
- Bug fix for displayed minimum temp when in metric

### CODE REVIEW

File: src/app/treasure-hunt/convert-input-data.service.ts [hunk 1]
```diff
@@ -332,6 +337,61 @@ export class ConvertInputDataService {
     return airLeakSurveys;
   }
 
+  convertFlueGasLosses(flueGasLosses: Array<FlueGasTreasureHunt>, oldSettings: Settings): Array<FlueGasTreasureHunt> {
+    flueGasLosses.forEach(flueGas => {
+      if (flueGas.baseline.flueGasType == 'By Volume') {
+        flueGas.baseline.flueGasByVolume = this.convertFlueGasInput(flueGas.baseline.flueGasByVolume, oldSettings);
+        flueGas.modification.flueGasByVolume = this.convertFlueGasInput(flueGas.modification.flueGasByVolume, oldSettings);
```
Comment (rmroot) [inline src/app/treasure-hunt/convert-input-data.service.ts:344]: I think this is a problem in the other areas but it's not guranteed a modification exists. Should add a check for all

File: src/app/treasure-hunt/convert-input-data.service.ts [hunk 2]
```diff
@@ -332,6 +337,61 @@ export class ConvertInputDataService {
...
+        flueGas.baseline.flueGasByMass = this.convertFlueGasInput(flueGas.baseline.flueGasByMass, oldSettings);
+        flueGas.modification.flueGasByMass = this.convertFlueGasInput(flueGas.modification.flueGasByMass, oldSettings);
+      }
+    });
+    return flueGasLosses;
+  }
+
+  convertFlueGasInput(flueGasInput: FlueGasByMass | FlueGasByVolume, currentSettings: Settings): FlueGasByMass | FlueGasByVolume {
+    if (currentSettings.unitsOfMeasure == 'Imperial'){
+      flueGasInput.combustionAirTemperature = this.convertUnitsService.value(flueGasInput.combustionAirTemperature).from('F').to('C');
```
Comment (rmroot) [inline src/app/treasure-hunt/convert-input-data.service.ts:356]: use the conversion functions at the bottom convertTemperatureValue(...)

**Review by rmroot — CHANGES_REQUESTED** [review]
Need to update import-opportunities.service.ts with flue gas (looks like export is there).

Test it by exporting your data from the treasure chest, creating a new treasure hunt and importing these opportunities

**Comment by nbintertech** [comment]
Closing for new branch after large TH changes

**Pull request closed without merging**